tests/src/actor_test.ts

```diff
@@ -90,8 +90,11 @@ export function startConnectedCndAndNectar(
     testFn: (actors: { alice: CndActor; bob: NectarActor }) => Promise<void>
 ): ProvidesCallback {
     return async (done) => {
-        const alice = await newCndActor("Alice", true);
-        const bob = await newNectarActor();
+        const startingAlice = newCndActor("Alice", true);
+        const startingBob = newNectarActor();
+
+        const alice = await startingAlice;
+        const bob = await startingBob;
```

Comment:
```suggestion
        const [alice, bob] = await Promise.all([newCndActor("Alice", true), newNectarActor()]);
```
nitpick/individual taste.

Reply:
I haven't tried it but I think that doesn't work because they are different types? Wouldn't TS get confused after this because both `alice` and `bob` are now of type `CndActor | NectarActor`?

Reply:
🤷🏻‍♂️